charts/gateway/templates/deployment.yaml

```diff
@@ -309,6 +408,26 @@ spec:
       {{- end }}
      {{- end }}
 {{- end }}
+{{- if .Values.existingHealthCheck }}
+{{- if .Values.existingHealthCheck.enabled }}
+     {{- if .Values.existingHealthCheck.configMap }}
+        - name: {{ .Values.existingHealthCheck.configMap.name }}
+          configMap:
+            defaultMode: {{ .Values.existingHealthCheck.configMap.defaultMode | default 292 }}
```

Comment:
Can an octal value be provided here? I'm assuming yes based on https://kubernetes.io/docs/concepts/configuration/secret/#secret-files-permissions. Thanks.

Reply:
> Can an octal value be provided here? I'm assuming yes based on https://kubernetes.io/docs/concepts/configuration/secret/#secret-files-permissions. Thanks.

Yes it can